### PR TITLE
OPENEUROPA-3058: Set event type widget to select.

### PR DESCRIPTION
**Pull request by nagyad**

### CODE REVIEW

File: oe_content.post_update.php [hunk 1]
```diff
@@ -129,3 +130,24 @@ function oe_content_post_update_00004(): void {
...
   }
 }
+
+/**
+ * Change widget for "Event type" field to skos select in form display.
+ */
+function oe_content_post_update_00005() {
+  if (!\Drupal::moduleHandler()->moduleExists('oe_content_event')) {
+    return 'Module oe_content_event is not enabled.';
+  }
```
Comment (ademarco) [inline oe_content.post_update.php:140]: Can't we just move this in `oe_content_event`? So it's ran only if that module is enabled?

File: composer.json [hunk 2]
```diff
@@ -78,6 +78,9 @@
             "drupal/typed_link": {
                 "https://www.drupal.org/project/typed_link/issues/3085826": "https://www.drupal.org/files/issues/2019-10-04/typed_link-3085826-2.patch",
                 "https://www.drupal.org/project/typed_link/issues/3085817": "https://www.drupal.org/files/issues/2019-10-07/typed_link-3085817-3.patch"
+            },
+            "openeuropa/rdf_skos": {
+                "latest-master": "https://github.com/openeuropa/rdf_skos/compare/0.6.0...master.diff"
```
Comment (ademarco) [inline composer.json:83]: Let's maybe release rdf_skos too as part of this ticket? So we don't have to do this.

File: oe_content.post_update.php (hunk 1 above)
Comment (nagyad) [reply oe_content.post_update.php:140]: Normally we prefer to place upgrade paths to the main module to ensure order. I thought I keep consistency.

File: composer.json (hunk 2 above)
Comment (nagyad) [reply composer.json:83]: Alright, I do it.

File: oe_content.post_update.php (hunk 1 above)
Comment (ademarco) [reply oe_content.post_update.php:140]: This code belongs to `oe_content_event`, which depends on `oe_content`, we have no order issues here.

File: modules/oe_content_event/oe_content_event.post_update.php [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+/**
+ * Change widget for "Event type" field to skos select in form display.
+ */
+function oe_content_event_post_update_00001() {
+  $display = EntityFormDisplay::load('node.oe_event.default');
+  $content = $display->get('content');
+
+  if (empty($content['oe_event_type'])) {
+    return 'Event type field is not exposed on the form display.';
```
Comment (upchuk) [inline modules/oe_content_event/oe_content_event.post_update.php:20]: Seeing that we are doing an update path here in the component, it means we are fixing something everywhere. So how come we are only fixing the default entity form display? What if there are more form displays that use the old select widget?

Comment (ademarco) [reply modules/oe_content_event/oe_content_event.post_update.php:20]: We could also only do this on EWCMS, we have followed this rationale in other occasions too. @brummbar what do you think?

Comment (brummbar) [reply modules/oe_content_event/oe_content_event.post_update.php:20]: This doesn't fix a bug indeed, so we should not have an opinion anymore on what's the best widget for the user that installed this module. Let's move this in EWPP.

**Pull request merged**